test/ctia/flows/hooks/kafka_event_hook_test.clj

```diff
@@ -22,15 +22,18 @@
                   es-helpers/fixture-properties:es-store
                   test-helpers/fixture-properties:kafka-hook
                   test-helpers/fixture-properties:events-enabled
-                  test-helpers/fixture-ctia
-                  test-helpers/fixture-allow-all-auth]))
+                  test-helpers/fixture-allow-all-auth
+                  test-helpers/fixture-ctia]))
```

Comment:
Need to switch the order of fixtures to prepare for TK -- `fixture-ctia` will start an app, and must go last to pick up all config changes.